packages/relay/tests/lib/eth.spec.ts

```diff
@@ -1681,35 +1681,6 @@ describe('Eth calls using MirrorNode', async function () {
       expectLogData(res, defaultLogs.logs[3], defaultDetailedContractResults3);
     };
 
-    it('blockHash filter timeouts and throws the expected error', async function () {
```

Comment:
We should still throw a timeout error.
If not we're not handling the timeout correctly.
What happens instead now?

Reply:
Test restored.